client/tck/src/test/java/test/unit/ConfigBehavior.java

```diff
@@ -102,37 +97,6 @@ void shouldLoadAnnotatedKeyConfig() {
         }
     }
 
-    static class DummyClientRequestFilter implements ClientRequestFilter {
-        @Override
-        public void filter(ClientRequestContext requestContext) {
-        }
-    }
-
-    @Test
-    void shouldRegisterClientRequestFilterClass() {
-        fixture.returnsData("'foo':true");
-        Api api = ((JaxRsTypesafeGraphQLClientBuilder) fixture.builder())
-                .register(DummyClientRequestFilter.class)
-                .build(Api.class);
-
-        api.foo();
-
-        verify(fixture.client()).register(DummyClientRequestFilter.class);
-    }
-
-    @Test
-    void shouldRegisterClientRequestFilterInstance() {
-        fixture.returnsData("'foo':true");
-        DummyClientRequestFilter instance = new DummyClientRequestFilter();
-        Api api = ((JaxRsTypesafeGraphQLClientBuilder) fixture.builder())
-                .register(instance)
-                .build(Api.class);
-
-        api.foo();
-
-        verify(fixture.client()).register(instance);
-    }
-
```

Comment:
These tests don't make any sense any more, as this is not part of the public API any more